CONTRIBUTING.md

```diff
@@ -0,0 +1,182 @@
+# 🤝 Contributing to octasol
+
+Welcome to Octasol! We are glad about your interest in contributing. Please take a look at this document to support an easy and effective contribution process for everyone.
+
+Following these guidelines shows respect for our maintainer's time, and in turn, they will respond to your issues and evaluate your contributions.
+
+> [!NOTE] 
+> Please submit one Pull Request per issue! 
+
+<p>&nbsp;</p>
+
+## 👀 Table of Contents
+1. [Setting Up Local Development Environment](#setting-up-local-development-environment)
+1. [Pull Requests Process](#pull-requests-process)
+1. [Code Guidelines](#code-guidelines)
+1. [Test your Code](#test-your-code)
+1. [How to Report Bugs](#how-to-report-bugs)
+1. [Your Help is needed](#your-help-is-needed)
+
+<p>&nbsp;</p>
+
+## Setting Up Local Development Environment
+
+
+1. Fork the Repository
+1. Clone the Repository
+
+    ```
+    git clone https://github.com/<yourusername>/octasol.git <octasol>
+
+    cd <octasol>
+    ```
+
+
+1. Install Dependencies
+
+    ```
+    npm install
+    ```
+1. Start the Development Server
+
+    ```
+    npm run dev
+    ```
+1. Set Up Environment Variables
+
+    Prisma uses an environment file (.env) for database connection details. Create your own .env file.
+    
+    Add your database connection string:
+
+    ```
+    DATABASE_URL="postgresql://USER:PASSWORD@HOST:PORT/DATABASE"
+    ```
+
+1. Migrate the Prisma Database
+
+    After setting up your .env file with the correct database connection, you’ll need to run Prisma migrations to set up the local database schema.
+
+    ```
+    npx prisma migrate dev
+    ```
+1. Generate Prisma Client
+
+    If it's not automatically generated during the installation or migration process, run:
+
+    ```
+    npx prisma generate
+    ```
+
+1. Start the Development Server
+
+    ```
+    npm run dev
+    ```
+
+1. Keep your repo up-to-date
+
+> [!NOTE] 
+> Keep your `master` branch pointing at the original repository and make
+> pull requests from branches on your fork. To do this, run:
+>
+> ```
+> git remote add upstream https://github.com/Octasol/octasol.git
+> git fetch upstream
+> git branch --set-upstream-to=upstream/master master
+> ```
+>
+> This adds the original repository as a "remote" called "upstream," fetches the latest changes from that repo, 
+>and sets your local `master` to track the upstream `master` branch. To update your `master` branch, run `git pull`. 
+>Afterward, create new branches from `master` for your pull requests.
+
+<p>&nbsp;</p>
+
+## Pull Requests Process
+
+1. Fork the Repository
+1. Clone the Repository
+
+     ```
+    git clone https://github.com/<yourusername>/octasol.git <octasol>
```

Comment:
please remove < and >

Reply:
Thanks for the feedback. I pushed a new commit considering your feedback